### ISSUE TITLE
Automation Validation - Deploy, Destroy, and Redeploy Platform Services Azure Cluster 

### ISSUE THREAD

**Issue by stewartshea** [BCDevOps/OpenShift4-RollOut#29]

**Comment by filiy** [comment]
This is happening. Just need to finalize the post deploy automated steps

**Comment by filiy** [comment]
Re-Deplopy 4.3 and testing automation steps again.

To do:
- oauth with github not working - need to check oauth on github
 -- adding yaml didn't work also, need to test again
- Infra machinesets. Remove worker label?? 
https://docs.openshift.com/container-platform/4.3/machine_management/creating-infrastructure-machinesets.html
- Disabling project self-provisioning. Seems like the autoupdate bit needs to be done first?
https://docs.openshift.com/container-platform/4.3/applications/projects/configuring-project-creation.html
- set up metering

**Comment by StevenBarre** [comment]
Hey @filiy I've got some of my cluster setup steps at https://github.com/bcgov-c/advsol-docs/blob/ocp-4-docs/OCP-4-Install.md#configure-github-auth if that helps. I had Shelly make me the github oauth keys.

> Infra machinesets. Remove worker label??
I did

**Comment by filiy** [comment]
Nice. Thanks @sbarre-esit !
 
Also need to figure this one out. The command the suggest doesn't work. This is for removing self project provisioning 
```
[phil@c3po terraform]$ oc --kubeconfig ../install-dir/auth/kubeconfig adm policy remove-cluster-role-from-group self-provisioner system:authenticated:oauth
Warning: Your changes may get lost whenever a master is restarted, unless you prevent reconciliation of this rolebinding using the following command: oc annotate clusterrolebinding.rbac self-provisioners 'rbac.authorization.kubernetes.io/autoupdate=false' --overwritecluster role "self-provisioner" removed: "system:authenticated:oauth"
```

**Comment by filiy** [comment]
done. Need to do a final test of post-install config but that is tracked in another issue.
